<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zchxRadarChannelReportCtrlDlg</class>
 <widget class="QDialog" name="zchxRadarChannelReportCtrlDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	max-height:20px;
	font-weight:normal;	
	max-width:40px;
}

#resetRadar,#video_output_start_btn{
	max-width:60px;
}

#common_frame,#skipVideoframe{
	background:transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="common_frame">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="Rang_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>扫描半径    </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="zchxRadarCtrlBtn" name="range_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="Interference_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>干扰抑制</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="zchxRadarCtrlBtn" name="injection_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Local_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>本地抗干扰</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="zchxRadarCtrlBtn" name="local_injection_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="boost_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>目标加速</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="zchxRadarCtrlBtn" name="target_boost_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="separation_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>目标分离  </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="zchxRadarCtrlBtn" name="target_separat_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="Sea_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>海杂波  </string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="zchxRadarCtrlBtn" name="sea_culter_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="zchxRadarCtrlBtn" name="sidelobe_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="Noise_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>噪声抑制</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Power_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>雷达电源</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="zchxRadarCtrlBtn" name="power_statsu_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="speed_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>扫描速度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Antenna_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>天线高度  </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="zchxRadarCtrlBtn" name="antenna_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="zchxRadarCtrlBtn" name="scan_seed_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Bearing_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>方位校准</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="zchxRadarCtrlBtn" name="bearing_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="expansion_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>目标扩展  </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Side_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>旁瓣抑制  </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="zchxRadarCtrlBtn" name="target_expansion_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="zchxRadarCtrlBtn" name="noise_injection_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="zchxRadarCtrlBtn" name="gain_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="Gain_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>增益    </string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="radius">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="factor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="radar_type">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="zchxRadarCtrlBtn" name="rain_culter_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="Rain_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>雨杂波  </string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>雷达类型</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="k_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>回波半径</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>距离因子</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="resetRadar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>重置雷达</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="video_output_start_btn">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>打印回波</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="skipVideoframe">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="fan1">
        <property name="text">
         <string>匿形1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>角度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="angle1">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="width3">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="fan3">
        <property name="text">
         <string>匿形3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="angle3">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="width2">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>角度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="fan4">
        <property name="text">
         <string>匿形4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="angle4">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="width4">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>角度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="width1">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>角度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="fan2">
        <property name="text">
         <string>匿形2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="angle2">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>zchxRadarCtrlBtn</class>
   <extends>QPushButton</extends>
   <header>zchxradarctrlbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
